frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts

```diff
@@ -204,6 +205,15 @@ export const playerSettingsLogic = kea<playerSettingsLogicType>([
                 setPrefersAdvancedFilters: (_, { prefersAdvancedFilters }) => prefersAdvancedFilters,
             },
         ],
+        quickFilterProperties: [
+            ['$geoip_country_name'] as string[],
+            {
+                persist: true,
+            },
+            {
+                setQuickFilterProperties: (_, { properties }) => properties,
+            },
+        ],
```

Comment:
this is really cool!

i can see the next question being "i set up my quick filters then they don't show on my desktop" :p

but that's the overlap with Ben's saved filters suggestion